### DISCUSSION TITLE
Capturer, Stériliser et Relâcher des chats errants.

### DISCUSSION THREAD

**Discussion by evefevrier** [integration2/quizTIM2023#48] [Q&A]
Commentaires sur ce quiz.

**Accès**
- Serveur de développement (protégé) : https://github.com/kateundercoffler/quiz-tim  
- Dépôt Github (privé pour le moment): https://timunix3.csfoy.ca/~quiz-TIM/kateundercoffler/index.html
- Github.io : https://kateundercoffler.github.io/quiz-tim/kateundercoffler/index.html

**Comment by Eli7792** [comment]
Le quiz est parfaitement fonctionnel et contient même des animations, wow ! Cependant, il y a quelques fautes de français à corriger.
